google/cloud/forseti/services/cli.py

```diff
@@ -163,6 +169,10 @@ def define_inventory_parser(parent):
     Args:
         parent (argparser): Parent parser to hook into.
     """
+
+    if parent is None:
+        LOGGER.warn("parent is None")
```

Comment:
This module uses both single quotes and double quotes around text inconsistently. We should use the same quote type everywhere in a module unless there is a specific case not to, for example 'This isn\'t right' would be better as "This isn't right" even if the rest of the code using single quotes.

I know this file was already messy before you modified it, but it is probably worth cleaning up in the cl.

https://google.github.io/styleguide/pyguide.html#Strings

Reply:
I agreed it's better to keep the style consistent. Can we open a separate issue just for cleaning up the codes?

Reply:
SGTM.